src/ngx_http_lua_kong_log.c

```diff
@@ -64,28 +64,62 @@ ngx_http_lua_kong_ffi_set_dynamic_log_level(int log_level, int timeout)
     }
 
     g_dynamic_log_level = log_level;
-    g_dynamic_log_level_timeout_at = (time_t)ngx_time() + (time_t)timeout;
+    g_dynamic_log_level_timeout_at = (time_t) ngx_time() + (time_t) timeout;
 
     return NGX_OK;
 }
 
+
 ngx_uint_t
 ngx_http_lua_kong_get_dynamic_log_level(ngx_uint_t current_log_level)
```

Comment:
btw It seems that no one will call this function, we could remove it in the later release.

Reply:
No, it will be called in a nginx patch (in kong's code base, build/openresty/patches.)

Reply:
This is called in Nginx core: https://github.com/Kong/kong/blob/1714e26e033ee7b0d296739149c8b19e2d96dc0e/build/openresty/patches/nginx-1.25.3_06-dynamic_log_level.patch#L11